.markdownlint.json

```diff
@@ -0,0 +1,10 @@
+{
+    "MD006": false,
+    "MD007": {"start_indented": true, "indent": 2, "start_indent": 2},
```

Comment:
I personally don't like `start_indent: 2`

Reply:
Do you prefer no indentation (=standard markdown)? I know that with this, I'm moving away from standard markdown, which is perhaps not the best idea...

Reply:
yes, I would keep it standard where ever possible, unless there is a good reason for it